src/main/java/com/samsolutions/employeesdep/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,30 @@
+package com.samsolutions.employeesdep.exception;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.util.Locale;
+
+@ControllerAdvice
+public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
+    @ExceptionHandler(EmployeesdepGlobalException.class)
+    @SuppressWarnings("rawtypes")
```

Comment:
Change ResponseEntity  to appropriate ResponseEntity<WHICH TYPE?>

Reply:
ResponseEntity<String> ist OK?

Reply:
ResponseEntity<String>